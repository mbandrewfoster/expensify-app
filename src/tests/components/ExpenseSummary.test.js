import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';
//import expenses from '../fixtures/expenses';

test('should render ExpenseSummary with 2 expenses', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={2} expensesTotal={50} />);
  expect(wrapper).toMatchSnapshot();
});

test('should render ExpenseSummary with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={590} />);
  expect(wrapper).toMatchSnapshot();
});