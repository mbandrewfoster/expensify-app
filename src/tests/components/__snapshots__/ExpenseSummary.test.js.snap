// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $5.90
  </h1>
</div>
`;

exports[`should render ExpenseSummary with 2 expenses 1`] = `
<div>
  <h1>
    Viewing 
    2
     
    expenses
     totalling 
    $0.50
  </h1>
</div>
`;
